### DISCUSSION TITLE
Notelinks dont work

### DISCUSSION THREAD

**Discussion by Kbentekik** [akosbalasko/yarle#481] [Q&A] [answered]
Trying to convert 1000 Evernote notes to Obsidian.

-I have 5 notebooks and use a lot of note-links.
-After conversion some note links work. But most of the links dont work.
-If i click on the notelink there is a new empty note created.
-All notes seem to be avaiable, when i search.
-If i right click on the notelinks which dont work: I see [[NotebookExample1/TheNote.md|TheNote]]
 But in reality the corresponding note is in NotebookExample2
-I did export the TOC of all my notes in a sepperate .enex. But this didn't help.
 When i did this notes are pointing to [[Table of contents/TheNote.md|TheNote]]
 Or do i have to create a table of content for each notebook?

Really hope i'm doing something wrong.

Thnx in advance.

**Comment by akosbalasko** [comment, marked as answer]
Hi @Kbentekik ,

Yes please generate TOC in every Notebook you want to convert. Don't create a separated enex for the TOC, Yarle requires to have TOC within the correct notebook. So the process would be the following:
1. Open a notebook in Evernote. 
2. Select all the notes
3. Click to generate Table of Contents (it will place the TOC not within the notebook)
4. Export the notebook including the generated TOC as an enex. 
5. Repeat the previous steps for each notebook you want to convert
6. Add all of the notebooks in Yarle's first step
7. Do the conversion. 

Hope it helps, please contact me if it still does not work.

**Comment by Kbentekik** [comment]
Solved.

A little more complicated.
-Making table of contents in Evernote Legacy (in v10 max 100 notes)
-Exporting in V10 (otherwise the tasks are not exported in good order)

Will take my personall notes for a testdrive in Obsidian.
Later my work-notes.

Thnx a lot!

![afbeelding](https://github.com/akosbalasko/yarle/assets/20271002/85fb4748-f27a-4aef-9de9-73bd5b2c9df9)
